test/browser/index.js

```diff
@@ -1,5 +1,5 @@
-const { createHashHistory, createHistory } = require('history');
+const { createHashHistory, createBrowserHistory } = require('history');
 const createTests = require('../createTests.js');
 
 createTests(createHashHistory, 'Hash History', () => window.location = '#/');
-createTests(createHistory, 'Browser History', () => window.history.replaceState(null, null, '/'));
+createTests(createBrowserHistory, 'Browser History', () => window.history.pushState(null, null, '/'));
```

Comment:
Changing `createHistory` to `createBrowserHistory` breaks the tests. History does not export `createBrowserHistory` so it is `undefined`.